api/services/okta_service.py

```diff
@@ -535,4 +535,16 @@ def update_okta_group(
 
 
 def is_managed_group(group: Group, group_ids_with_group_rules: dict[str, list[OktaGroupRuleType]]) -> bool:
+    # Check if 'allow_discord_access' attribute exists as a custom Okta Group Profile attribute and retrieve its value
+    allow_discord_access = getattr(group.profile, "allow_discord_access", None)
```

Comment:
Not to bikeshed too much on the attribute name, but could we either name it something like `managed_by_access` or `access_is_authoritative`? Or better yet, could we have the attribute name be customizable with an environment variable (added [here](https://github.com/discord/access/blob/main/api/config.py)) in case other deployments already have an attribute they want to use?

Reply:
Made some adjustments on my local.  Just trying to get my tests to work, but new ENV VAR for this sounds fine to me.

Reply:
Added a new optional parameter to `is_managed_group` so that I can mock it in the added tests.  Seems to work fine and all the tests pass. Spot checking on my local, functionality of the override works too.  

Resolving this conversation but welcoming further comments if any.

Reply:
New ENV VAR is `OKTA_GROUP_PROFILE_CUSTOM_ATTR`